src/run/run.c

```diff
@@ -1054,6 +1061,16 @@ static int parse_argv_sudo_mode(int argc, char *argv[]) {
         if (!l)
                 return log_oom();
 
+        if (arg_switch_shell) {
+                arg_exec_path = strdup("/bin/sh");
```

Comment:
hmm, not following here, shouldn't there be "-c" added to the cmdline if this is used?

Reply:
This is just a placeholder, and will be normalized by pid1 anyways. `-c` will be added implicitly in exec_invoke()